Sources/Plasma/PubUtilLib/plResMgr/plResManager.cpp

```diff
@@ -1330,25 +1335,11 @@ bool plResManager::VerifyPages()
     }
 
     // Step 2 of verification: make sure no sequence numbers conflict
-    // This isn't possible with a std::map
-    /*PageSet::iterator it = fAllPages.begin();
-    for (; it != fAllPages.end(); it++)
-    {
-        plRegistryPageNode* page = *it;
-
-        PageSet::iterator itUp = it;
-        itUp++;
-        for (; itUp != fAllPages.end(); itUp++)
-        {
-            plRegistryPageNode* upPage = *itUp;
-            if (page->GetPageInfo().GetLocation() == upPage->GetPageInfo().GetLocation())
-            {
-                invalidPages.Append(upPage);
-                fAllPages.erase(itUp);
-                break;
-            }
-        }
-    }*/
+    for (auto badPage : fConflictingPages) {
+        fAllPages.erase(badPage->GetPageInfo().GetLocation());
+        invalidPages.Append(badPage);
```

Comment:
If I'm reading the old code correctly, it only removed (and added to invalidPages) the second (and later) copies of the page; your change here appears to remove and track both the duplicate and the original...

Reply:
That seems to be correct. Removing both (or more) was a purposeful design decision on my part however. It is better to wipe away all of the potentially invalid pages and let the client download the correct one when linking to the Age. Otherwise, you can potentially get stuck in a spin-wash cycle where the old logic would delete the new page and not touch the old one, requiring a redownload on each link. The inverse could happen as well, but that's a best-case scenario...

Reply:
Makes sense.